lib/tooltip.js

```diff
@@ -67,6 +67,31 @@ class Tooltip {
     this.config = config;
     this.step = step;
     this.tutorial = tutorial;
+
+    config = Object.assign({
+      xOffsetTooltip: 0,
+      yOffsetTooltip: 0,
+      offsetArrow: 0,
+      attr: {},
+      arrowLength: DEFAULT_ARROW_LENGTH,
+      subtext: (() => `${this.currentStepNum()} of ${this.tutorial.steps.length}`)
+    }, config);
+
+    if (!config.position) {
+      throw new Error('tooltop position must be defined');
+    }
+    const selectorKeys = Object.keys(this.step.selectors);
+    if (selectorKeys.length > 1 && !config.anchorElement) {
+      throw new Error('anchorElement is not optional when more than one ' +
+        'selector exists:\n' + this);
+    } else if (typeof config.subtext !== 'function') {
+      throw new Error('subtext must be a function')
+    }
+
+    config = Object.assign({
+      anchorElement: selectorKeys[0]
```

Comment:
I suppose a user could pass in `[undefined]` or `{ key: undefined }` to the `selectors` argument, so I could do some checking for that pathological edge case in `Step`'s constructor.